crates/nu-command/Cargo.toml

```diff
@@ -87,9 +87,10 @@ toml = "0.8"
 unicode-segmentation = "1.10"
 ureq = { version = "2.8", default-features = false, features = ["charset", "gzip", "json", "native-tls"] }
 url = "2.2"
-uu_cp = "0.0.22"
-uu_whoami = "0.0.22"
-uu_mkdir = "0.0.22"
+uu_cp = { git = "https://github.com/uutils/coreutils.git", package = "uu_cp" }
+uu_whoami = { git = "https://github.com/uutils/coreutils.git", package = "uu_whoami" }
+uu_mkdir = { git = "https://github.com/uutils/coreutils.git", package = "uu_mkdir" }
```

Comment:
not quite sure why you changed uu_cp, uu_whoami, and uu_mkdir to point to the git repo. 0.0.22 was working for these already. I can understand uu_mktemp if there are changes being made though, but I'm not positive there are.

Reply:
What I'm saying is the first 3 were already working on 0.0.22. I can understand uu_mktemp not working without a release, but the others were I believe. It's not that big of a deal right now because we can't land it this way anyway.

Reply:
Oh right yeah I had to update coreutils https://github.com/uutils/coreutils/pull/5479.

Reply:
Right, that means you need to point uu_mktemp to the git dependency, not the other commands. That's my point.